src/sagemaker/jumpstart/model.py

```diff
@@ -312,6 +314,27 @@ def _is_valid_model_id_hook():
 
         super(JumpStartModel, self).__init__(**model_init_kwargs.to_kwargs_dict())
 
+    def retrieve_default_payload(self) -> JumpStartSerializablePayload:
```

Comment:
sorry I missed this: this instance method needs be renamed `retrieve_example_payload()`

Please also add a `retrieve_all_example_payloads()` instance method.